app/views/publications/monthly_statistics/show.html.erb

```diff
@@ -269,7 +275,23 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l govuk-!-padding-top-4" id="footnotes">12. Footnotes</h2>
+    <h2 class="govuk-heading-l govuk-!-padding-top-4" id="downloads">12. Download the data</h2>
+
+    <p class="govuk-body">You can download all of the data on this page in CSV (comma separated values) format, which can be opened in a spreadsheet and used in other analysis tools.</p>
+
+    <p class="govuk-body">The downloads include some further breakdowns of the data by combining fields - for example, applicant age and sex.</p>
+
+    <% @report.each do |export_type, size| %>
+      <p class='govuk-body'>
+    <%= govuk_link_to t("#{export_type}.label") + " #{size.to_s(:human_size)}", publications_monthly_report_download_path(export_type: export_type, date: Time.zone.today.strftime('%Y-%m')) %>
```

Comment:
```suggestion
      <%= govuk_link_to t("#{export_type}.label") + " #{size.to_s(:human_size)}", publications_monthly_report_download_path(export_type: export_type, date: Time.zone.today.strftime('%Y-%m')) %>
```

Reply:
Better to hardcode `Time.zone.today.strftime('%Y-%m')` to `2021-11` for now, until we have the data changes in (ie when we know which month's report we're after)?

Reply:
thanks - changes made